xchange-core/src/main/java/org/knowm/xchange/dto/account/Balance.java

```diff
@@ -275,26 +278,32 @@ public BigDecimal getDepositing() {
     return depositing;
   }
 
+  /**
+   * Returns the timestamp of the balance
```

Comment:
the is no sense in repeating the word timestamp in the docs

should the client set this field when the data was downloaded or should this information be from the exchanges server?

Reply:
fixed